### ISSUE TITLE
Add boolean option to normalize images before computing likelihoods

### ISSUE THREAD

**Issue by mjo22** [mjo22/cryojax#243]
In the `cryojax.distributions.IndependentGaussianFourierModes` class, add a boolean option to normalize simulated images to mean zero and standard deviation 1.

If the `AbstractImagingPipeline` class has a mask, do this normalization only where the mask is equal to 1 with `jax.numpy.mean(…, where=…)` and `jax.numpy.std(…, where=…)`. The mask can be used to define the region of signal in the simulated image.

**Comment by mjo22** [comment]
Closed with #244
